sys/include/net/netopt.h

```diff
@@ -539,6 +539,85 @@ typedef enum {
      */
     NETOPT_BLE_CTX,
 
+    /**
+     * @brief   (uint8_t*) device EUI (8 bytes length)
+     */
+    NETOPT_DEVEUI,
+
+    /**
+     * @brief   (uint8_t*) device EUI (4 bytes length)
+     */
+    NETOPT_DEVADDR,
+
+    /**
+     * @brief   (uint8_t*) application EUI (8 bytes length)
+     */
+    NETOPT_APPEUI,
+
+    /**
+     * @brief   (uint8_t*) application key (16 bytes length)
+     */
+    NETOPT_APPKEY,
```

Comment:
we have a NETOPT_ENCRYPTION, but can be confusing. Maybe adding a LORAWAN prefix to these LoRaWAN specific NETOPTS? (or a LPWAN prefix if applies)